### DISCUSSION TITLE
Notebook 00 discrepancy in the instructions vs. autograder

### DISCUSSION THREAD

**Discussion by taylor-obrien** [earthlab-education/Earth-Analytics-2023-01-Intro#292] [Q&A03 Python] [answered]
Elsa,

In the notebook 00 for this weeks assignment, in the part titled "Create List of Converted Monthly Temperatures", it asks to make a list of jan-jun, but the autograder says this is incorrect. I tried a list with only jan-mar and that gives full points in the autograder. Then at the end of the autograder, it says "you have received points for making a list jan-jun", despite it only giving you points if you make a list for jan-mar. Please help!

**Comment by edchanonline** [comment, marked as answer]
Agreed the instructions were not clear with the months. I created a list `[jan, feb, mar]` for this exercise to pass the autograder because those were the only data provided in Table 1. I believe it was also said somewhere I should expect three variables only (but then it said to expect six variables later somewhere else). In the subsequent exercise, I then appended/extended the list with values from April to December (despite the instrunction is asking for July through December.)

**Reply by taylor-obrien** [reply]
Okay great, I did the same! Thanks for the reply!

**Reply by eculler** [reply]
Sorry about that! It used to be jan-june, but I shifted it to more playing with lists. I'll correct that for next time!
